doc/zephyr.doxyfile.in

```diff
@@ -220,8 +220,8 @@ TAB_SIZE               = 8
 # "Side Effects:". You can put \n's in the value part of an alias to insert
 # newlines.
 
-ALIASES                = "rst=\verbatim embed:rst:leading-asterisk" \
-                         "endrst=\endverbatim"
+ALIASES  = "rst=\verbatim embed:rst:leading-asterisk" "endrst=\endverbatim"
+ALIASES += "prop=\note @b Properties: @b"
```

Comment:
Should the alias be named `funcprop` instead? That is what we decided here:
https://github.com/zephyrproject-rtos/zephyr/issues/21061#issuecomment-627455133

Reply:
why do you need func in here? we have param, not funcparam, brief, not funcbrief! I think @prop should be enough and function is implied.

Reply:
Doxygen also already has [@property](http://www.doxygen.nl/manual/commands.html#cmdproperty) which does something different and could confuse people.

Reply:
I think we should use a unequivocal name that is exclusive to zephyr to avoid any potential clashes. Hence the `funcprop` proposal that we settled on last week.